exonum/src/runtime/mod.rs

```diff
@@ -609,6 +609,17 @@ impl<'a> ExecutionContext<'a> {
         }
     }
 
+    /// Returns extensions required for the Supervisor service implementation.
+    ///
+    /// Make sure that this method invokes by the instance with the [`SUPERVISOR_INSTANCE_ID`]
```

Comment:
Grammar: ~invokes~ is invoked by.

Also, I don't quite like this kind of guarantees. Would it make sense to leave parts (or all) of `CallContext` in the main crate? IMO, there's nothing inherently Rustic about `CallContext`, and `InstanceDescriptor` necessary for its function can be obtained from `Dispatcher`, if I'm not mistaken.

Reply:
https://jira.bf.local/browse/ECR-4075